mfr/extensions/pdf/render.py

```diff
@@ -13,7 +15,19 @@ class PdfRenderer(extension.BaseRenderer):
         ]).get_template('viewer.mako')
 
     def render(self):
-        return self.TEMPLATE.render(base=self.assets_url, url=self.metadata.download_url)
+        if self.metadata.ext in settings.EXPORT_EXCLUSIONS:
+            return self.TEMPLATE.render(base=self.assets_url, url=self.metadata.download_url)
+
+        exported_url = furl.furl(self.export_url)
+        if settings.EXPORT_MAXIMUM_SIZE and settings.EXPORT_TYPE:
```

Comment:
This reduce the # of `bool` checks from 3 to 2.
```python
if settings.EXPORT_TYPE:
    if settings.EXPORT_MAXIMUM_SIZE
        exported_url.args['format'] = '{}.{}'.format(settings.EXPORT_MAXIMUM_SIZE, settings.EXPORT_TYPE)
    else:
        exported_url.args['format'] = settings.EXPORT_TYPE
    self.metrics.add('needs_export', True)
    return self.TEMPLATE.render(base=self.assets_url, url=exported_url.url)

# TODO: is this an unexpected case?
return self.TEMPLATE.render(base=self.assets_url, url=self.metadata.download_url)
```

Reply:
This was mostly copied from the image renderer. I assume that the TODO case here is when the settings are not properly defined. Going with your approach here but leaving the unexpected case in for completion etc.

Reply:
Agreed. Similarly, update image renderer if possible.